lib/spells/effects/Dispel.cpp

```diff
@@ -102,9 +102,7 @@ bool Dispel::mainSelector(const Bonus * bonus)
 		if(sourceSpell->id == SpellID::CLONE)
 			return false;
 		//stack may have inherited effects
-		if(sourceSpell->isAdventureSpell())
-			return false;
-		return true;
+		return !sourceSpell->isAdventureSpell();
```

Comment:
Use ```CSpell::isCombatSpell```.

Reply:
Yep good catch, like I write above such simplification it's good way to make code more expressive.